packages/fxa-auth-server/test/oauth/api.js

```diff
@@ -2861,7 +2863,7 @@ describe('#integration - /v1', function () {
     it('should not reject expired tokens from pocket clients', async function () {
       const clientId = '749818d3f2e7857f';
       config.set('oauthServer.expiration.accessTokenExpiryEpoch', undefined);
-      Server = await testServer.start();
+      Server2 = await testServer.start();
```

Comment:
If I'm understanding this correctly, we starting another server here because the new server instance needs to be started with the updated config from line 2865?

Reply:
That's what it seemed like to me.